src/ExpenseItemList.tsx

```diff
@@ -1,39 +1,36 @@
 import React, { ChangeEvent } from 'react';
-import ExpenseItem from "./ExpenseItem";
+import ExpenseItem from './ExpenseItem';
```

Comment:
Seems like our linters let different things fly.

Reply:
double quotes are the devil